pkg/asset/cluster/tfvars.go

```diff
@@ -199,6 +199,11 @@ func (t *TerraformVariables) Generate(parents asset.Parents) error {
 			}
 		}
 
+		ignitionBucket, ignitionPresignedURL, err := installConfig.AWS.IgnitionPresignedURL(ctx, clusterID.InfraID)
+		if err != nil {
+			return err
+		}
+
```

Comment:
maybe we can modify it to do the following?

```
url, err := presignedURL(installConfig.AWS.GetSession(), installConfig.AWS.Region,  bucketname, objectname)
```

Reply:
I've updated the function to work this way.